src/main/java/edu/kis/powp/jobs2d/drivers/adapter/BaseAdapter.java

```diff
@@ -4,17 +4,27 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+import javax.swing.*;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class BaseAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name does not indicate the purpose of the adapter or what was actually adapted.
Inheritance is not used (no super drawLine calls). Should be removed!